app/commands/parse_markdown.rb

```diff
@@ -32,18 +32,18 @@ def sanitized_html
 
   memoize
   def raw_html
-    renderer = Renderer.new(options: [:UNSAFE], nofollow_links: nofollow_links)
-    html = CommonMarker.render_doc(
+    doc = CommonMarker.render_doc(
       preprocessed_text,
       :DEFAULT,
       %i[table tagfilter strikethrough]
     )
-    renderer.render(html)
+    renderer = Renderer.new(options: [:UNSAFE], nofollow_links: nofollow_links)
+    renderer.render(doc)
   end
 
   memoize
   def preprocessed_text
-    text.gsub(/^`{3,}(.*?)`{3,}\s*$/m) { "\n#{$&}\n" }
+    text.gsub(/^`{3,}(.*?)`{3,}\s*$/m) { "\n#{Regexp.last_match(0)}\n" }
```

Comment:
This is not me, this must be Rubocop.

Reply:
`Regexp.last_match(0)` is preferred over `$&`.

`$&` is a [special global variable](https://docs.ruby-lang.org/en/2.5.0/Regexp.html#class-Regexp-label-Special+global+variables) so you "must know" what it is.

Reply:
Ah, thanks